tests/ident/test_engine_parser_msfragger_3.py

```diff
@@ -140,10 +140,10 @@ def test_engine_parsers_msfragger_unify_row():
             "15N": False,
         },
     )
-    for row in parser:
-        assert row["Sequence"] == "ATTALTDDTLDGAGR"
-        assert row["Search Engine"] == "msfragger_3_0"
-        break
+    df = parser.unify()
+    first_row = df.iloc[0, :]
+    assert first_row["Sequence"] == "ATTALTDDTLDGAGR"
+    assert first_row["Search Engine"] == "msfragger_3_0"
```

Comment:
same as above - please assert identity of the resulting df - not just first line :)